tests/test_cleanup_functions.py

```diff
@@ -25,6 +25,14 @@ def test_cleanup_fileref():
         assert bug_features.cleanup_fileref(orig_text) == cleaned_text
 
 
+def test_cleanup_hex():
+    tests = [
+        ('0 scdetour.dll scdetour.dll@0x2dd77', '0 scdetour.dll __HEX_NUMBER__'),
```

Comment:
There seems to be something wrong in the result of the test, it should be `0 scdetour.dll scdetour.dll@__HEX_NUMBER`.

Also add a word which is just an hex number in the test.